.github/workflows/jekyll.yml

```diff
@@ -1,20 +1,81 @@
 name: Jekyll site CI
-
 on:
-  push:
-    branches: [ changes, release, translate ]
   pull_request:
-    branches: [ changes, release, translate ]
-
+    branches: [ next-release, release ]
+  push:
+    branches: [ next-release ]
 jobs:
-  build:
+  Jekyll_site_CI:
     permissions:
-      contents: read
+      contents: write
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Build the site in the jekyll/builder container
-      run: |
-        docker run \
-        -v ${{ github.workspace }}:/srv/jekyll -v ${{ github.workspace }}/_site:/srv/jekyll/_site \
-        jekyll/builder:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 777 /srv/jekyll && jekyll build --future"
+      - uses: actions/checkout@v2
+
+# Retrieve po4a installation cache or create it if not found:
+      - name: Cache po4a dependencies
+        uses: actions/cache@v1.0.3
+        id: cache-po4a
+        with:
+          path: "~/po4a"
+          key: ${{ runner.os }}-po4a
+      - name: Install or retrieve po4a from cache
+        env:
+          CACHE_HIT: ${{steps.cache-po4a.outputs.cache-hit}}
+        run: |
+          if [[ "$CACHE_HIT" == 'true' ]]; then
+            sudo cp --force --recursive ~/po4a/* /
+          else
+            sudo apt install -yq gettext libsgmls-perl libyaml-tiny-perl opensp
+            wget -O po4a.deb https://github.com/jamulussoftware/assets/raw/main/po4a/po4a_0.64-alpha.deb
+            sudo dpkg -i po4a.deb
+            mkdir -p ~/po4a
+            for dep in po4a libcroco3 libosp5 sgml-base gettext libsgmls-perl libyaml-tiny-perl opensp; do
+                dpkg -L $dep | while IFS= read -r f; do if test -f $f; then echo $f; fi; done | xargs cp --parents --target-directory ~/po4a/
+            done
+          fi
+
+# Paths changes filter. If there's been a change to any files defined in the filter, the step (update templates) runs:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          base: ${{ github.ref }}
+          filters: |
+            templates:
+              - 'wiki/en/*.md'
+      - name: Update templates
+        if: steps.filter.outputs.templates == 'true'
+        run: ./_po4a-tools/po4a-update-templates.sh
+# This step only runs if a PR is merged:
+      - name: Push changes to repo if action triggered on:push and templates need updating
+        if: ${{ github.event_name == 'push' && steps.filter.outputs.templates == 'true' }}
+        run: |
+            git config --local user.name "ignotus666"
```

Comment:
This will have to change

Reply:
Yes. And we should probably use another way to commit changes. There should be an official way to add commits by a GH actions step.

Reply:
Done.